src/Entity/OgMembership.php

```diff
@@ -59,23 +59,46 @@
  *   fieldable = TRUE,
  *   bundle_entity_type = "og_membership_type",
  *   entity_keys = {
+ *     "uuid" = "uuid",
```

Comment:
This seems to introduce #425 and will require a hook_update in order for the key to be added to the schema.

Reply:
Maybe we can do this change in a separate issue, and also do the hook_update?

Reply:
@jludwig, see #537.

Could you add me as an author on jludwig:og-membership-ui?

Reply:
@MPParsley I just added you as a collaborator.

Reply:
Thanks!